CS342MemberViews/Package.swift

```diff
@@ -17,9 +17,13 @@ let package = Package(
         .iOS(.v16)
     ],
     products: [
-        .library(name: "PaulSchmiedmayer", targets: ["PaulSchmiedmayer"])
+        .library(name: "PaulSchmiedmayer", targets: ["PaulSchmiedmayer"]),
+        .library(name: "AudreyLin", targets: ["AudreyLin"])
     ],
     targets: [
+        .target(
+            name: "StudentViews"
+        ),
```

Comment:
Hey @ivyalin2015! The files for `StudentView` seem to be missing from this branch and the project isn't compiling. Are you sure you added them to your commit before you pushed to Github?